setup/features/server-group-launch-settings/aws-ec2/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: single
+title:  "Overview"
+sidebar:
+  nav: setup
+---
+
+## Overview
+An AWS EC2 Server Group offers a number of launch setting configurations that can enhance your experience.
+
+### Launch Settings
+An AWS Server Group can be set up with two types of launch settings. However, <b>you must enable Launch Templates support for all of your applications if you want to use the latest AWS features.
+AWS strongly recommends using Launch Templates over Launch Configurations because Launch Configurations do NOT provide full functionality for Amazon EC2 Auto Scaling or Amazon EC2.</b>
+
+1. (Older) [launch configuration](https://docs.aws.amazon.com/autoscaling/ec2/userguide/LaunchConfiguration.html) is an instance configuration template that an AWS Auto Scaling group uses to launch EC2 instances.
+2. (Newer) [launch template](https://docs.aws.amazon.com/autoscaling/ec2/userguide/LaunchTemplates.html) is similar to a launch configuration, in that it specifies instance configuration information.
+   However, defining a launch template instead of a launch configuration allows you to:
```

Comment:
Preferred style: "However, by defining a launch template instead of a launch configuration, you can:" (we prefer that to "allows you to")